src/fiatconnect/clients.test.tsx

```diff
@@ -50,6 +51,10 @@ describe('getFiatConnectClient', () => {
     const fcClient = await getFiatConnectClient('provider1', 'https://provider1.url')
     expect(getWalletAsync).toHaveBeenCalled()
     expect(fcClient).toBeInstanceOf(FiatConnectClient)
+    expect(fcClient).toHaveProperty(
+      'config.timeout',
+      REMOTE_CONFIG_VALUES_DEFAULTS.fiatConnectTimeoutSeconds * 1000
+    )
```

Comment:
This test depends on the implementation detail of the FiatConnectClient. Open to suggestions on an easy way to test this without this dependency

Reply:
this seems okay to me, you could `jest.mock` the value but i don't see much of a benefit.